.github/workflows/ci.yml

```
@@ -226,6 +226,45 @@ jobs:
+  #
+  # Keep the feature list in sync with cd.yml's macOS rows, the same way the
+  # `all-sources` leg tracks its Linux row.
+  macos:
+    name: macOS (check + clippy)
+    runs-on: macos-latest
+    needs: prepare
+    env:
+      FEATURES: "telemetry,tui,streaming,discord-rpc,cover-art,self-update,scripting,mcp-server,ai-dj,audio-viz-cpal,portaudio-backend,macos-media,local-files,subsonic,internet-radio,youtube,qobuz"
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)